ffmpeg/nvidia_test.go

```diff
@@ -787,6 +787,11 @@ func TestTranscoder_Portrait(t *testing.T) {
 	// Error should be `profile 250x62500 size out of bounds 146x146-4096x4096 input=16x4000 adjusted 250x62500 or 16x4096`
 }
 
+func TestNvidia_Resolution(t *testing.T) {
+	runResolutionTests_H264(t, Nvidia)
+	// TODO HEVC clamping
```

Comment:
Let's raise a ticket for this if we're not doing it as part of this PR

Reply:
https://linear.app/livepeer/issue/ENG-2258/add-hevc-resolution-tests